src/runner.js

```diff
@@ -170,7 +189,13 @@ class Runner {
       }
 
       const optionResults = new results.OptionsResult(filteredSelections.map((s) => {
-        return s.text;
+        const formatInstance = new MessageFormat(
+          this.options.formaterOptions.language || 'en' // Fallback language to English
```

Comment:
you are already setting it to 'en' on ln24, so perhaps || 'en' is redundant here?

Reply:
U're right, I'll change that